pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -8,13 +8,19 @@ import (
 	"github.com/code-ready/crc/pkg/crc/adminhelper"
 	"github.com/code-ready/crc/pkg/crc/cluster"
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/daemonclient"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/machine/bundle"
 	"github.com/code-ready/crc/pkg/crc/preset"
 	"github.com/code-ready/crc/pkg/crc/validation"
+	"github.com/code-ready/crc/pkg/crc/version"
 	"github.com/pkg/errors"
 )
 
+// This make sure golint doesn't error out because not
+// using it on all the platforms
+var _ = isDaemonRunningWithReleasedVersion()
```

Comment:
`isDaemonRunningWithReleasedVersion` is almost the same code as `checkDaemonStarted` which is used on all platforms, so this could be factored.

Reply:
@cfergeau Handled in #3099 , will hold this until that merge and then rebase it